Assets/Scripts/Presenters/InputPresenter.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using System.Collections.Generic;
+using Enums;
+using Models;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Presenters
+{
+    public class InputPresenter : MonoBehaviour
+    {
+        [SerializeField] private GameObject newOfferButton;
+        [SerializeField] private GameObject inputFallback;
+        [SerializeField] private TMP_Dropdown offeringResourceType;
+        [SerializeField] private TMP_InputField offeringResourceAmount;
+        [SerializeField] private TMP_Dropdown requestingResourceType;
+        [SerializeField] private TMP_InputField requestingResourceAmount;
+        [SerializeField] private TMP_Dropdown targetTribe;
+        [SerializeField] private TMP_Text errorText;
+        private TradePresenter _tradePresenter;
+
+        private void Start()
+        {
+           _tradePresenter = GetComponent<TradePresenter>();
+           targetTribe.ClearOptions();
+           targetTribe.AddOptions(new List<string>(){GameManager.Instance.Cpu1.Name,GameManager.Instance.Cpu2.Name});
+        }
+
+        public void ToggleNewOfferButton(bool isActive)
+        {
+            newOfferButton.SetActive(isActive);
+        }
+        public void ToggleInputFallBack()
+        {
+            inputFallback.SetActive(!inputFallback.activeSelf);
+            newOfferButton.SetActive(!inputFallback.activeSelf);
```

Comment:
When the Trade UI is shown on screen, I can still see the new Offer button? Isn't that supposed to be the counter offer button? Or why is there a new Offer button visible there?